packages/ui/src/machines/authenticator/__tests__/signIn.test.ts

```diff
@@ -0,0 +1,13 @@
+import { createAuthenticatorMachine } from '../index';
+
+describe('createAuthenticatorMachine', () => {
+  it('should create AuthenticatorMachine and run actor when sign-in', () => {
+    const authenticatorMachine = createAuthenticatorMachine();
+    const expectedValue = { signIn: 'runActor' };
+    const actualState = authenticatorMachine.transition('signIn', {
+      type: 'SUBMIT',
+    });
+
+    expect(actualState.matches(expectedValue)).toBeTruthy();
```

Comment:
Prefer checking for actual values over truthiness

Reply:
I know! I wish Xstate could do that. Or maybe I missed something, but that's in Xstate's [guide](https://xstate.js.org/docs/guides/testing.html#mocking-effects)

Reply:
Removed the test 0d9d07f19f1ddd26804daa5e6865a4845fb58bd3